### PR TITLE
feat(billing-counter) Removed iframe billing-counter callbacks

### PR DESCRIPTION
**Pull request by robertpin**

### CODE REVIEW

**Comment by quitrk** [comment]
We should move remaining stuff into features/jaas, either now, or in a future PR

**Comment by robertpin** [comment]
> We should move remaining stuff into features/jaas, either now, or in a future PR

Done

File: react/features/jaas/functions.js [hunk 1]
```diff
@@ -1,9 +1,88 @@
...
+
+/**
+ * Returns true if the current meeting is a vpaas one.
+ *
+ * @param {Object} state - The state of the app.
+ * @param {boolean} requiredJwt - Whether jwt is required or not.
+ * @returns {boolean}
+ */
+export function isVpaasMeeting(state: Object, requiredJwt: boolean = true) {
+    const { billingCounterUrl, iAmRecorder, iAmSipGateway } = state['features/base/config'];
```
Comment (quitrk) [inline react/features/jaas/functions.js:43]: billingCounterUrl should not be a condition anymore

File: react/features/jaas/functions.js [hunk 2]
```diff
@@ -1,9 +1,88 @@
...
+    return billingId;
+}
+
+/**
+ * Returns the billing id for vpaas meetings.
+ *
+ * @param {Object} state - The state of the app.
+ * @returns {string | undefined}
+ */
+export function getVpaasBillingId(state: Object) {
```
Comment (quitrk) [inline react/features/jaas/functions.js:80]: are these two used anymore?

Comment (robertpin) [reply react/features/jaas/functions.js:79]: They're used in the conference options.
image

File: react/features/jaas/functions.js [hunk 3]
```diff
@@ -1,9 +1,87 @@
...
+import { BILLING_ID, VPAAS_TENANT_PREFIX } from './constants';
 import logger from './logger';
 
+/**
+ * Returns the full vpaas tenant if available, given a path.
+ *
+ * @param {string} path - The meeting url path.
+ * @returns {string}
+ */
+export function extractVpaasTenantFromPath(path: string) {
```
Comment (quitrk) [inline react/features/jaas/functions.js:15]: looks like wherever this is used externally, it does exactly what `getVpaasTenant` is doing. I would remove the export and replace the usages to `getVpaasTenant`

File: react/features/jaas/functions.js (hunk 2 above)
Comment (quitrk) [reply react/features/jaas/functions.js:79]: By the looks of [this commit](https://github.com/jitsi/jitsi-meet/commit/bfc4b2ac6f0ce0431ede825e38fec1673ab00280) it's a leftover of the previous implementation. @vp8x8  please confirm whether this is the case and if so, whether it is safe to remove the code

Comment (vp8x8) [reply react/features/jaas/functions.js:79]: Yes, this is correct. It can be removed now.

Comment (robertpin) [reply react/features/jaas/functions.js]: Removed it

**Comment by quitrk** [comment]
please rebase and you'll probably have to update the path here https://github.com/jitsi/jitsi-meet/commit/0ab905bf756e8bac6287f629c040cec62717716e#diff-17c94552d943a928a39aea6f4358eaebfccd678b252cf36572a94361bf9030e4R6

**Review by quitrk — DISMISSED** [review]

**Review by quitrk — DISMISSED** [review]

**Comment by quitrk** [comment]
Jenkins please test this please

**Comment by quitrk** [comment]
Jenkins please test this please

**Comment by quitrk** [comment]
Jenkins please test this please

**Review by quitrk — APPROVED** [review]

**Pull request merged**